.github/workflows/workflow-executor.yaml

```diff
@@ -8,6 +8,11 @@ on:
         default: latest
         required: false
         type: string
+      auto_upgrade:
```

Comment:
Do we think it will potentially be confusing that we have a speakeasy_version config here.

But we will actually be respecting the speakeasyVersion in workflow.yaml

Reply:
If a speakeasyVersion doesn't exist in workflow.yaml and auto_update is not set will we still use the latest version to stay backward compatible with what we have now